lua/config/net_conf.lua

```diff
@@ -0,0 +1,7 @@
+--TODO Edit the network configuration data.
```

Comment:
Please add a space after `--`. Although I'm commenting only this line, please do it for all Lua comments.